infrastructure/asp.tf

```diff
@@ -0,0 +1,20 @@
+locals {
+
+  ase_name = "core-compute-${var.env}"
+  common_tags = {
+  }
+}
+
+
+
+module "appServicePlan" {
```

Comment:
Do you not want to put this in a shared infrastructure repository rather than having it in the citizen-frontend repo?

Reply:
just doing that now. but other wise is everything ok?

Reply:
https://github.com/hmcts/cmc-shared-infrastructure/pull/8